### PR TITLE
[TEST] [E2E] Delete shipping method from the shipping zone details page

### PR DESCRIPTION
**Pull request by yellowee**
## What type of PR is this?
- [ ] 💅 Refactor
- [ ] 🌟 Feature
- [ ] 🔥 Bug Fix
- [x] 🔩 Maintenance
- [ ] 🛠 Workflow CI/CD changes

## Have you written tests?
- [ ] Yes!
- [x] No... here is why: the pr introduces new test scenario

## [Optional] Description

It covers [TC SALEOR_34](https://saleor.testmo.net/repositories/1?group_id=1193&case_id=7558)

### CODE REVIEW

File: playwright/pages/dialogs/deleteShippingMethodDialog.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+export class DeleteShippingMethodDialog {
+  readonly page: Page;
+
+  readonly deleteButton: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+
+    this.deleteButton = page.getByTestId("submit");
+  }
```
Comment (wojteknowacki) [inline playwright/pages/dialogs/deleteShippingMethodDialog.ts:12]:
```suggestion
  readonly page: Page;

  constructor(page: Page, readonly deleteButton = page.getByTestId("submit")) {
    this.page = page;
  }
```

File: playwright/tests/shippingMethods.spec.ts [hunk 2]
```diff
@@ -27,8 +27,8 @@ test("TC: SALEOR_31 Create basic shipping method @shipping-method @e2e", async (
   await shippingMethodsPage.basePage.expectSuccessBanner();
 });
 test("TC: SALEOR_32 Add price rate to shipping method - with excluded zip codes adn excluded product @shipping-method @e2e", async ({
```
Comment (michalina-graczyk) [inline playwright/tests/shippingMethods.spec.ts:29]: adn -> and

File: playwright/pages/shippingMethodsPage.ts [hunk 3]
```diff
@@ -67,6 +83,7 @@ export class ShippingMethodsPage {
       timeout: 10000,
```
Comment (michalina-graczyk) [inline playwright/pages/shippingMethodsPage.ts:83]: is it necessary?

File: playwright/tests/shippingMethods.spec.ts [hunk 4]
```diff
@@ -56,8 +56,8 @@ test("TC: SALEOR_32 Add price rate to shipping method - with excluded zip codes
   );
 });
 test("TC: SALEOR_33 Add weight rate to shipping method - with included zip codes adn excluded product @shipping-method @e2e", async ({
```
Comment (michalina-graczyk) [inline playwright/tests/shippingMethods.spec.ts:58]: adn -> and

**Review by michalina-graczyk — CHANGES_REQUESTED** [review]

File: playwright/tests/shippingMethods.spec.ts (hunk 2 above)
Comment (yellowee) [reply playwright/tests/shippingMethods.spec.ts]: fixed

File: playwright/tests/shippingMethods.spec.ts (hunk 4 above)
Comment (yellowee) [reply playwright/tests/shippingMethods.spec.ts]: fixed

File: playwright/pages/dialogs/deleteShippingMethodDialog.ts (hunk 1 above)
Comment (wojteknowacki) [inline playwright/pages/dialogs/deleteShippingMethodDialog.ts:12]:
```suggestion
  readonly deleteButton: Locator;

  constructor(page: Page, readonly deleteButton = page.getByTestId("submit")) {
    this.page = page;
  }
```

File: playwright/tests/shippingMethods.spec.ts [hunk 5]
```diff
@@ -85,3 +85,21 @@ test("TC: SALEOR_33 Add weight rate to shipping method - with included zip codes
...
+test("TC: SALEOR_34 Delete a single shipping method from the shipping zone details page @shipping-method @e2e", async ({
+                                                                                                                         page,
+                                                                                                                       }) => {
+  const shippingMethodsPage = new ShippingMethodsPage(page);
+  await shippingMethodsPage.gotoExistingShippingMethod(
+    SHIPPING_METHODS.shippingMethodWithRatesToBeDeleted.id,
+  );
+  await expect(shippingMethodsPage.basePage.pageHeader).toBeVisible();
+  const priceBasedRate = SHIPPING_METHODS.shippingMethodWithRatesToBeDeleted.rates.priceBasedRateToBeDeleted.name;
+  await expect(shippingMethodsPage.priceBasedRatesSection).toContainText(priceBasedRate);
```
Comment (wojteknowacki) [inline playwright/tests/shippingMethods.spec.ts:98]: Do you need it?

**Review by andrzejewsky — DISMISSED** [review]
Seems like your test is not working + please fix the CI checks, eg. ts check

**Review by wojteknowacki — APPROVED** [review]

**Review by andrzejewsky — APPROVED** [review]

**Pull request merged**